tests/admin_changelist/tests.py

```diff
@@ -1595,6 +1595,29 @@ def test_search_role(self):
             '<form id="changelist-search" method="get" role="search">',
         )
 
+    def test_navigation_role(self):
```

Comment:
I'm not sure if need `test_navigation_role` as this is already covered by other tests.

Reply:
That's a good point @felixxm, we do have the other tests which have been modified as well. Should I just remove ```test_navigation_role```?

Reply:
> Should I just remove test_navigation_role?

I think it's fine.

Reply:
Ok, thanks!